ibexa/commerce/5.0/encore/ibexa.webpack.config.js

```diff
@@ -5,6 +5,10 @@ const configSetups = require('./var/encore/ibexa.config.setup.js');
 const path = require('path');
 
 module.exports = (Encore) => {
+    process.env.NODE_ENV =
+        process.env.NODE_ENV ||
+        (Encore.isProduction() ? 'production' : 'development');
```

Comment:
could use, up to you, cleaner code, if you know what this operator does ;)
```suggestion
    process.env.NODE_ENV ??= Encore.isProduction() ? 'production' : 'development';
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment